cli/tests/integrations/test_service.py

```diff
@@ -99,28 +99,21 @@ def test_service_inactive_and_completed():
 
 
 def test_log():
-    with package(
-        'cassandra',
-        deploy=True,
-        args=['--package-version=1.0.25-3.0.10']
-    ):
+    with package('hello-world', deploy=True):
         returncode, stdout, stderr = exec_command(
-            ['dcos', 'service', 'log', 'cassandra'])
+            ['dcos', 'service', 'log', 'hello-world'])
```

Comment:
One could consider parameterising the app name?

Reply:
you mean having a variable on top? makes sense, I'll do that in a follow-up PR, I can keep a `minor` branch around and push it from time to time.